labs/intro.md

```diff
@@ -1,111 +1,61 @@
-## Lab exercise 0: Setup steps
+<a id="markdown-accessing-lab-environment" name="accessing-lab-environment"></a>
+# Accessing to the Lab Environment
 
-:imagesdir: images
+## Getting your dedicated lab environments using Guid Grabber
 
-### Accessing your dedicated lab environment using your provided GUID
+On your laptop, open **three** browser tabs to [*Lab GUID Assignment page*](https://www.opentlc.com/gg/gg.cgi?profile=generic_tester).
 
-On your laptop, browse to https://www.opentlc.com/gg/gg.cgi?profile=generic_summit[*Lab GUID Assignment page*^].
-From this page, you will be assigned your unique GUID, which you will use to access your unique lab environment and systems.
-
-Select the proper *Lab Code* from the drop down list. Find the lab `XXXXXXX - Hands on with Red Hat Multi-Cluster Federation: Application Portability`
-The activation key is *XXXXXXX*.
-
-The resulting *Lab Information page* will display your lab's GUID and other useful information about your lab environment.
-Take note of your assigned GUID.
-You will use this GUID to access your lab's environment and systems.
-Your unique GUID will also be embedded in your lab environment's host names.
-From this *Lab Information page*, you will also be able to access your unique lab environment's power control and consoles.
-Go to the last bullet point and click on *here* to access the environment's power control and consoles.
-
-image:labinfopage.png[1200,1200]
-
-Notice that one workstation VM is shown on the power control and consoles page.
-This is a Red Hat Enterprise Linux 8 system with GUI and will be the machine that we will use throughout all the lab exercises in this lab.
-Click the *CONSOLE* button.
-Log in as Lab User with password *r3dh4t1!*.
-
-image:vmconsole.png[200,200]
-
-Congratulations, you got to your *graphical console*.
-
-
-## Using the text console to access the remote shell
-
-On your laptop (i.e. not in the browser window with the remote deskop), open the `Terminal` application.
-Then, enter the following command, replacing `GUID` with your actual GUID:
+### CHANGEME URL for RHTE will be generic_rhte
```

Comment:
Do you need to change this now before PR?

Reply:
Nope, we need to change it before moving the lab to GA.